script.js

```diff
@@ -1,8 +1,45 @@
+/* eslint-disable no-unused-vars */
```

Comment:
- Hey, let's remove this eslint disable and try to fix the linter by writing code that conforms with it to improve your development skill.

Reply:
I already removed eslint disable comment and requested for a new review